packages/headless/src/features/facets/facet-search-set/generic/generic-facet-search-actions.ts

```diff
@@ -46,6 +50,16 @@ export const executeFacetSearch = createAsyncThunk<
   }
 );
 
+/**
+ * Resets the query and empties the values of the facet search.
+ * @param facetId (string) The unique identifier of the facet for which to perform a facet search (e.g., `"1"`).
+ */
+export const clearFacetSearch = createAction(
+  'facetSearch/ClearResults',
```

Comment:
I think all segments of actions we have today with a lower-case? Using `facetSearch/clearResults` would be more consistent.